### DISCUSSION TITLE
Data sources for possible tests

### DISCUSSION THREAD

**Discussion by jgaffuri** [eurostat/gridviz#117] [Ideas]
NL: https://www.cbs.nl/nl-nl/dossier/nederland-regionaal/geografische-data/kaart-van-100-meter-bij-100-meter-met-statistieken
World: https://wopr.worldpop.org/?NGA/Population/v2.0
https://www.pdok.nl/introductie/-/article/cbs-vierkantstatistieken-100m
AT: https://www.statistik.at/atlas/?mapid=them_bevoelkerung_bevoelkerungsstand&layerid=layer1&sublayerid=sublayer0&languageid=1
https://www.statistik.at/en/services/tools/services/regional-information/regional-statistics
250m population by year... not free. free: 10km sample. Citizenship/origin
IT: http://gisportal.istat.it/GriAnalyst
griglia di popolazione 2011 https://www.istat.it/it/archivio/155162
FI: https://kartta.paikkatietoikkuna.fi/?lang=en
1km. pop by gender and age, yearly since 2005.
http://www.stat.fi/org/avoindata/paikkatietoaineistot/vaestoruutuaineisto_1km_en.html
as WFS: https://geo.stat.fi/geoserver/vaestoruutu/wfs
SE: https://www.scb.se/en/services/open-data-api/open-geodata/grid-statistics/
IE: https://www.cso.ie/en/census/census2016reports/workplacezonesand1kmpopulationgrids/
EE: https://estat.stat.ee/StatistikaKaart/VKR
SI: https://gis.stat.si/#lang=en
UK-NI: irregular grid - with 100m. population https://www.nisra.gov.uk/support/geography/grid-square-products
NO:
Statistics Norway: https://kart.ssb.no/
For more grid statistics (Norwegian only): https://www.ssb.no/natur-og-miljo/areal/artikler/kart-og-geodata-fra-ssb
250m population by gender. many years.
Kart og geodata fra SSB (own homepage, search for «Nedlasting av rutenettstatistikk»)
Befolkningsstatistikk (POP) (250m / 1000m / 5000m) - Population statistics
Boligstatistikk (DWE) (250m / 1000m / 5000m) - Dwelling statistics
Bygningsmassen (BUI) (250m / 1000m / 5000m) - Construction statistics
Virksomhetsstatistikk (EST) (250m / 1000m / 5000m) - Enterprise statistics
Landbrukseiendommer (AGP) (1000m / 5000m) - Agricultural properties
Jordbruksstatistikk (AGH) (1000m / 5000m) - Agricultural holdings
Handels- og serviceruter/-soner (250m) - Commerce and service routes/zones

ideamapsnetwork.org - developing a gridded dataset on urban poverty
US Census Bureau International Database and Subnational Projections (for PEPFAR countries) has extensive time series data through 2100 as well - Samuel I Dupre (CENSUS/POP FED)
WorldPop - long time series
https://ghsl.jrc.ec.europa.eu/ESMVisualisation.php
https://ghsl.jrc.ec.europa.eu/enactVisualisation.php

DE
German Census 2011 data in 100m grid cells for population, housing, families and houeseholds
https://www.zensus2011.de/DE/Home/Aktuelles/DemografischeGrunddaten.html?nn=559100#Gitter
Agriculture (5 and 10km grid)
https://agraratlas.statistikportal.de/
Mobility indicators by mobile phone usage
https://storymaps.arcgis.com/stories/58b7b89db42749e69b611a4764870546
It looks like we have grid based tourism data for each year from 2017-2021, see
https://gis-hsl.hessen.de/portal/apps/webappviewer/index.html?id=621b2739ff8f43e895a759cc0dcc2c70
however it is available as WMS but it is not immediately clear if we offer the data in csv-Format as well and this afternoon is not a good time to find somebody who might know. But let me know if this is something, you might be interested in, even though it's far from a 10yr+ timeseries.

**Comment by jgaffuri** [comment]
check copernicus urban atlas

copernicus global land cover: https://land.copernicus.eu/global/products/lc

**Comment by jgaffuri** [comment]
ENACT dataset: https://ghsl.jrc.ec.europa.eu/ENACT_POP.php
With timeseries style.

**Comment by jgaffuri** [comment]
https://land.copernicus.eu/global/products/
https://land.copernicus.eu/pan-european/high-resolution-layers

**Comment by jgaffuri** [comment]
internet speed grid
OOKLA dataset - https://github.com/teamookla/ookla-open-data

**Reply by joewdavies** [reply]
I did a test a while back, could be improved though: https://joewdavies.github.io/gridviz-ookla/viewer/

**Reply by jgaffuri** [reply]
Great !
There is a little artfact when zooming in at maximum level - nothing severe - it might be due to some reprojection ?

**Comment by jgaffuri** [comment]
EDGAR JRS dataset
https://edgar.jrc.ec.europa.eu/emissions_data_and_maps

**Comment by jgaffuri** [comment]
JRC Drought dataset: https://edo.jrc.ec.europa.eu/edov2/php/index.php?id=1000

**Reply by jgaffuri** [reply]
https://edo.jrc.ec.europa.eu/tumbo/edo/map/

**Comment by jgaffuri** [comment]
agri4cast dataset:
https://agri4cast.jrc.ec.europa.eu/dataportal/

**Comment by jgaffuri** [comment]
Cover Crops accross Europe - European Soil Data Centre
https://esdac.jrc.ec.europa.eu/content/cover-crops-accross-europe#tabs-0-description=1

https://www.sciencedirect.com/science/article/pii/S0167880924002615

**Reply by jgaffuri** [reply]
https://data.jrc.ec.europa.eu/collection/id-00346

**Comment by jgaffuri** [comment]
climate/weather
https://www.worldclim.org/data/monthlywth.html
https://chelsa-climate.org/
https://prism.oregonstate.edu/

**Reply by jgaffuri** [reply]
ERA5-Land. Smoothed daily maximum temperature (1981-2021)
https://twitter.com/dr_xeo/status/1729438782552719844

**Comment by jgaffuri** [comment]
climate / air pollution
https://atmosphere.copernicus.eu/new-interface-our-european-air-quality-forecasts?utm_source=socialmedia&utm_medium=tw&utm_campaign=news-airquality-september
https://sdi.eea.europa.eu/catalogue/datahub/eng/catalog.search#/metadata/82700fbd-2953-467b-be0a-78a520c3a7ef

**Comment by jgaffuri** [comment]
https://www.esa-landcover-cci.org/

**Comment by jgaffuri** [comment]
HISDAC-ES: historical settlement data compilation for Spain (1900–2020)

https://essd.copernicus.org/articles/15/4713/2023/

**Comment by jgaffuri** [comment]
Forest fire risk
https://civio.es/espana-en-llamas/2023/10/17/el-territorio-de-europa-en-riesgo-alto-de-incendios-se-ha-duplicado-en-los-ultimos-50-anos/

**Comment by jgaffuri** [comment]
Crop map
https://data.jrc.ec.europa.eu/collection/id-00346
On NL and FR

**Comment by jgaffuri** [comment]
Global land temperature
https://halftone.co/projects/temperatures/https://www.ncei.noaa.gov/products/land-based-station/global-historical-climatology-network-monthly

**Comment by jgaffuri** [comment]
EU energy atlas:
https://joint-research-centre.ec.europa.eu/jrc-news-and-updates/high-resolution-energy-consumption-atlas-shape-future-eu-energy-infrastructures-2024-02-26_en

**Comment by jgaffuri** [comment]
https://www.ine.es/censos2011_datos/cen11_datos_resultados_rejillas.htm
Metadata: https://www.ine.es/censos2011_datos/indicadores_seccen_rejilla.xls

1km resolution.
2011 only.
Numerous breakdowns (age, sex, education level, etc.).

**Comment by jgaffuri** [comment]
https://www.data.gouv.fr/fr/datasets/donnees-changement-climatique-lsh-longues-series-homogeneisees/

**Comment by jgaffuri** [comment]
Mixed resolution grid, belgium:
https://statbel.fgov.be/fr/open-data/datalab-grid-de-population-cellules-de-taille-variable
https://statbel.fgov.be/fr/themes/datalab/grid-mailles-variables

**Comment by wahlatlas** [comment]
German Census 2022 in 10km, 1km and 100m resolution for
Demography, Household Size, Rent, Owner Occupied Share, Size of Dwelling, Heating & Energy and more

English 
https://www.zensus2022.de/EN/News/Hinweis_Zensusatlas_VOE_en.html
German 
https://www.zensus2022.de/DE/Aktuelles/Hinweis_Zensusatlas.html

**Comment by jgaffuri** [comment]
CZ: https://geodata.csu.gov.cz/as/georeporty/ext/
